.github/workflows/test-operator-wandb.yaml

```
@@ -49,11 +49,13 @@ jobs:
         configuration: [
+            "anaconda2",
             "default",
             "executor-enabled",
-            "user-defined-secrets",
+            # "runs-v2-bufstream",
             "separate-pods",
-            "runs-v2-bufstream",
+            "separate-pods-filemeta",
+            "user-defined-secrets",
             "weave-trace",
             "separate-pods-filemeta",
```

actionlint (1.7.7)

duplicate value "separate-pods-filemeta" is found in matrix "configuration". the same value is at line:57,col:13

(matrix)